ygen/gogen.go

```diff
@@ -382,6 +386,19 @@ func (s *{{ .StructName }}) Validate(opts ...ygot.ValidationOption) error {
 	}
 	return nil
 }
+`
+
+	// goContainerGetterTemplate defines a template that generates a getter function
+	// for the field of a generated struct. It is generated only for YANG containers.
+	goContainerGetterTemplate = `
+// Get{{ .Field.Name }} returns the value of the {{ .Field.Name }} struct pointer
```

Comment:
Should we be more specific and mention that this also happens to return nil in some cases?

Reply:
This was done in the last commit, thanks.